wgengine/wglog/wglog.go

```diff
@@ -85,7 +85,7 @@ func (x *Logger) SetPeers(peers []wgcfg.Peer) {
 	// Construct a new peer public key log rewriter.
 	replace := make(map[string]string)
 	for _, peer := range peers {
-		old := "peer(" + wireguardGoString(peer.PublicKey) + ")"
+		old := wireguardGoString(peer.PublicKey)
 		new := peer.PublicKey.ShortString()
```

Comment:
no need to call this if it exists in the old map still. Reuse the old map's value, now that you have the `old` key, you can look up `oldMap, _ := x.replace.Load.(map[string]string)` and `oldMap[old]`?

Reply:
Fair enough. LGTM as is.